test/test_overrides.py

```diff
@@ -782,15 +785,15 @@ class TestEinsumOverride(TestCase):
     def test_wrapper(self):
         x = Wrapper(torch.randn(5))
         y = Wrapper(torch.randn(4))
-        self.assertTrue(torch.allclose(torch.einsum('i,j->ij', x, y),
-                                       torch.ger(x, y)))
+        self.assertEqual(torch.einsum('i,j->ij', x, y)._data,
+                         torch.ger(x, y)._data)
```

Comment:
Not 100% sure if this tests only for value equality or not (cc @rgommers who contributed the test in #38741). If yes, we need the extra `._data` access, because due to the methods defined in `Wrapper`, `TestCase.assertEqual` takes the output as iterable and then fails because it can't get a length of it.

Reply:
Yes just testing the values is fine here.